easybuild/easyconfigs/p/PyQtGraph/PyQtGraph-0.11.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'PyQtGraph'
+version = '0.11.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pyqtgraph.org/'
+description = """PyQtGraph is a pure-python graphics and GUI library built on PyQt5/PySide2 and numpy."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+github_account = 'pyqtgraph'
+source_urls = [GITHUB_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['082cdec1e559644e083cd8c5752fc06a18582fb2bebb83cdf6eb8ad33c735535']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),  # required for numpy
+    ('PyQt5', '5.13.2', versionsuffix),
+    ('PyOpenGL', '3.1.5'),
+]
+
+use_pip = True
+download_dep_fail = True
```

Comment:
Same as above.

Reply:
May I ask why this is stalled?  Are here real issues?  I need the PyOpenGL for another build project.